awx/api/serializers.py

```diff
@@ -3469,7 +3471,9 @@ def get_related(self, obj):
         if 'last_job' in res:
             del res['last_job']
 
-        res.update(dict(jobs = self.reverse('api:workflow_approval_template_jobs_list', kwargs={'pk': obj.pk}),))
+        res.update(dict(
+            jobs = self.reverse('api:workflow_approval_template_jobs_list', kwargs={'pk': obj.pk}),
```

Comment:
the wrapper of `dict(...)` is extraneous here, `.update()` works with keyword arguments just as well as passing it a dict.